kvbc/src/ror_app_state.cpp

```diff
@@ -0,0 +1,160 @@
+#include "ror_app_state.hpp"
+#include <cassert>
+#include <cstring>
+#include <unordered_map>
+#include <vector>
+#include "hash_defs.h"
+
+using namespace concordUtils;
+using namespace bftEngine::SimpleBlockchainStateTransfer;
+using namespace concord::storage;
+
+class RoRAppStateImpl : public IAppState {
```

Comment:
I don't think we need the implementation derive from `IAppState`. It is an implementation detail that is defined in the cpp file and it can just have the same (or similar) interface, without inheriting from `IAppState`.

Reply:
I'm trying to remove the RoRAppstate at all right now